### PR TITLE
Metadata loss warning while renaming libraries

### PR DESCRIPTION
**Pull request by Artiume**
**Changes**

Add a warning on metadata loss while renaming libraries. Could've made a confirm window, went with using the label for less code.

**Issues**

https://www.reddit.com/r/jellyfin/comments/pk8evo/comment/ipqyk41/

### CODE REVIEW

File: src/controllers/dashboard/library.js [hunk 1]
```diff
@@ -76,7 +76,8 @@ import cardBuilder from '../../components/cardbuilder/cardBuilder';
     function renameVirtualFolder(page, virtualFolder) {
         import('../../components/prompt/prompt').then(({default: prompt}) => {
             prompt({
-                label: globalize.translate('LabelNewName'),
+                title: globalize.translate('LabelNewName'),
+                label: globalize.translate('MessageAreYouSureYouWishToRenameMediaFolder'),
```
Comment (thornbill) [inline src/controllers/dashboard/library.js:80]:
```suggestion
                label: globalize.translate('LabelNewName'),
                description: globalize.translate('MessageAreYouSureYouWishToRenameMediaFolder'),
```

**Review by thornbill — CHANGES_REQUESTED** [review]
I think this will work better. Title is the heading displayed next to the back arrow and label is the input field label.

Screenshot 2022-09-29 at 16-25-39 Jellyfin

File: src/controllers/dashboard/library.js (hunk 1 above)
Comment (nielsvanvelzen) [reply src/controllers/dashboard/library.js:80]: When I look at your screenshot, I don't think it should be a question but a note/warning instead. Something like "Renaming a folder clears all all metadata, proceed with caution".

File: src/strings/en-us.json [hunk 2]
```diff
@@ -1036,6 +1036,7 @@
     "MessageAlreadyInstalled": "This version is already installed.",
     "MessageAreYouSureDeleteSubtitles": "Are you sure you wish to delete this subtitle file?",
     "MessageAreYouSureYouWishToRemoveMediaFolder": "Are you sure you wish to remove this media folder?",
+    "MessageAreYouSureYouWishToRenameMediaFolder": "Are you sure you wish to rename this media folder? All metadata will be lost",
```
Comment (thornbill) [inline src/strings/en-us.json:1039]:
```suggestion
    "MessageAreYouSureYouWishToRenameMediaFolder": "Renaming a media library will cause all metadata to be lost, proceed with caution.",
```

**Review by crobibero — APPROVED** [review]

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**